common/cluster-api/src/main/proto/queue.proto

```diff
@@ -193,6 +193,18 @@ message GetOrCreateDeviceFromGatewayResponseMsg {
   bytes profileBody = 2;
 }
 
+message GetOrCreateDeviceFromSparkplugRequestMsg {
```

Comment:
We don't need this structure because it is a code duplication from the `GetOrCreateDeviceFromGatewayRequestMsg/GetOrCreateDeviceFromGatewayResponseMsg`.
Let's add the `boolean sparkplug` to the `GetOrCreateDeviceFromGatewayRequestMsg` instead.